### PR TITLE
Update Godeps etcd to 3.0.13

### PR DESCRIPTION
**Pull request by wojtek-t**
Ref #20504

---

This change is [Reviewable](https://reviewable.kubernetes.io/reviews/kubernetes/kubernetes/35694)

### CODE REVIEW

**Comment by wojtek-t** [comment]
@timothysc @xiang90 @hongchaodeng

**Comment by wojtek-t** [comment]
@k8s-bot verify test this, issue; #IGNORE (broken build)

**Comment by timothysc** [comment]
lgtm.

**Comment by wojtek-t** [comment]
@k8s-bot verify test this, issue; #IGNORE (broken head)

File: build-tools/build-image/cross/VERSION [hunk 1]
```diff
@@ -1 +1 @@
-v1.7.1-0
+v1.7.1-2
```
Comment (luxas) [inline build-tools/build-image/cross/VERSION]: just FYI @jessfraz, this will conflict with https://github.com/kubernetes/kubernetes/pull/35716

**Comment by jessfraz** [comment]
ah gotcha thanks

**Comment by wojtek-t** [comment]
@k8s-bot gce node e2e test this, issue: #34990

**Comment by wojtek-t** [comment]
@k8s-bot node e2e test this, issue: #34990

**Comment by luxas** [comment]
@wojtek-t Can you push the new image for all architectures please?

**Comment by wojtek-t** [comment]
@luxas - so far I didn't push it for any architecture, since it's still in the testing phase. I will push it for all architectures when I will be pushing it.

**Pull request merged**